docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -188,7 +133,7 @@ rendered, the next page of the archive is fetched. A list of the fetched pages i
 
```

Comment:
Change:
"This hook is more complex than the previous one, as it works getting the post type entities from the specified archive and thus it doesn't fetch the next post but the next page of posts."
to:
"This hook is more complex than the previous one, as it works by getting the post type entities from the specified archive and thus it doesn't fetch the next post but the next page of posts."